src/main/java/bio/terra/cda/app/controller/QueryApiController.java

```diff
@@ -95,24 +94,21 @@ private String createNextUrl(String jobId, int offset, int limit) {
     return sendQuery(querySql, dryRun);
   }
 
+  /**
+   * Table cells that represent as nested structures must be unnested from it decendant heiarchy.
```

Comment:
This duplicates the comment in `NestedColumn` so there's no need to put it here.

Reply:
Removed duplicate comments